package.json

```diff
@@ -11,6 +11,7 @@
     "precommit": "lint-staged"
   },
   "main": "src/screens.js",
+  "types": "src/screens.d.ts",
```

Comment:
We should add `src/screens.d.ts` to the "files" section below too in order for it to be included in the package sent to npm

Reply:
Ah, wait, we actually have whole "src/" there so this should not be necessary